src/pages/DetailsPage/__tests__/DetailsPage.test.tsx

```diff
@@ -68,7 +72,19 @@ const mockSingleProduct = (hasData: boolean) => {
     sku: hasData ? 'TESTSKU' : '',
     serviceLevel: hasData ? 'TEST serviceLevel' : '',
     serviceType: hasData ? 'TEST serviceType' : '',
-    unitOfMeasure: hasData ? { name: 'test', quantity: '2' } : null
+    unitOfMeasure: hasData ? { name: 'test', quantity: '2' } : null,
+    subscriptions: hasData
+      ? [
+          {
+            number: '1234',
+            contractNumber: '2345',
+            quantity: '1',
+            endDate: '2022-10-24T04:00:00.000Z',
+            status: 'Active',
+            startDate: '2021-10-24T04:00:00.000Z'
+          }
+        ]
+      : []
   };
 
```

Comment:
It won't let me comment below, but it might be worth adding tests to check if data is loading,  if there is an error, if quantity is missing for both the subscription details as well as the table. Seems coverage is a little low for this page

Reply:
I'm not sure why it is low here, but there is a test case for both loading and missing data. I can try adding an error state test and see if that increases coverage any

Reply:
Added an error test